streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorNode.java

```diff
@@ -214,9 +214,18 @@ public void process(final Record<KIn, VIn> record) {
                 internalProcessorContext.currentNode().name(),
                 internalProcessorContext.taskId());
 
-            final ProcessingExceptionHandler.ProcessingHandlerResponse response = processingExceptionHandler
-                .handle(errorHandlerContext, record, e);
-
+            final ProcessingExceptionHandler.ProcessingHandlerResponse response;
+
+            try {
+                response = processingExceptionHandler
+                        .handle(errorHandlerContext, record, e);
+            } catch (final Exception fatalUserException) {
+                log.error(
+                    "Processing error callback failed after processing error for record {}",
+                    record,
```

Comment:
We should not log records for privacy reasons. The exception should be enough.

Reply:
done